packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -370,8 +370,9 @@ exports.questions = {
 			new ConditionalRequiredValidator('Enter how many days you expect the inquiry to last'),
 			new StringValidator({
 				regex: {
-					regex: '^(\\s*|\\d+)$',
-					regexMessage: 'How many days you expect the inquiry to last must be a number, like 30'
+					regex: '^([1-9]|[1-9]\\d{1,2})$',
```

Comment:
`^\d{1,3}$`?

Reply:
Not too good with regex aha will try that

Reply:
That one accepts leading zeros, do you know how to make it from 1 to 999?

Reply:
ooh I see yah, closer to your original one then `^[1-9]{1,3}$` chars from 1 to 9 between 1 and 3 times, does that work?

Reply:
ah yeah this is awkward to validate as a string 😅

Reply:
`^[1-9]\d{0,2}$` just a smidge simpler without the `|`

<img width="202" alt="Screenshot 2023-11-13 at 11 21 03" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/fb2039aa-0bbc-4b6f-bd49-9b071cf566a8">

Reply:
Nice that works thanks, updated the code